apps/root/src/pages/dca/positions/components/positions-list/position-card/components/position-warning/index.tsx

```diff
@@ -62,27 +62,6 @@ const PositionWarning = ({ position }: PositionWarningProps) => {
         </Link>
       </>
     );
-  } else if (position.from.symbol === 'agEUR' || position.to.symbol === 'agEUR') {
-    message = (
-      <FormattedMessage
-        description="positionagEURNotSupported"
-        defaultMessage="Due to Euler's security breach, the Angle protocol has been paused. As a consequence, oracles and swaps cannot operate reliably and have been halted."
-      />
-    );
-  }
-
-  if ((!!position.from.underlyingTokens.length || !!position.to.underlyingTokens.length) && position.chainId === 1) {
-    message = (
-      <>
-        <FormattedMessage
```

Comment:
Why not just deprecate it? If the claim URL does not work, then what are we going to say? "Euler was hacked, you don't have a way of claiming". There is literally less than 400 USD in ALL Euler recovery smart contracts. Basically 1 user that will never be back.

Reply:
At least yeah, a minimum warning of why you cant increase/withdraw from your position

Reply:
I still think it's not worth having this check since:

- The check is naive af
- It will be impossible to help recover those funds
- There are 400 USD in ALL euler recovery SCs

But, I've re-added another message & modified it on localazy!